app/transport/http/const.go

```diff
@@ -0,0 +1,8 @@
+package http
+
+const (
+	MsgUserCreated      = "user successfully created"
+	MsgEmailConflict    = "email is already taken"
+	MsgInternalSeverErr = "internal server error"
+	MsgBadRequest       = "request is bad"
```

Comment:
Let's not invent the wheel. We have a well known "Bad request"

Reply:
Agreed